### PR TITLE
Add Bundler 1.15 release post

### PR DESCRIPTION
**Pull request by segiddins**
Would like to release tonight / tomorrow

\c @colby-swandale @indirect

### CODE REVIEW

File: source/blog/2017-05-19-bundler-1-15-bundle-oh-so-fast.html.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+date: 2017-05-19 12:00 UTC
+tags:
+author: Samuel Giddins
+author_url: http://segiddins.me
+category: release
+---
+
+# What’s new in Bundler 1.15?
+
+Hot on the heels of the many small fixed in Bundler 1.14, we're pushing out 1.15. The list of changes is much shorter, but we think you're going to love it all the same, since this time around we've focused on making Bundler a whole heck of a lot faster.
```
Comment (colby-swandale) [inline source/blog/2017-05-19-bundler-1-15-bundle-oh-so-fast.html.markdown]: Think you're missing `many small _bugs_ fixed`

Comment (segiddins) [inline source/blog/2017-05-19-bundler-1-15-bundle-oh-so-fast.html.markdown]: `many small fixes` is I think what t

Comment (segiddins) [inline source/blog/2017-05-19-bundler-1-15-bundle-oh-so-fast.html.markdown]: fixed

**Comment by segiddins** [comment]
Probably want to get https://github.com/bundler/bundler-site/pull/299 in as well, but I need to run. @indirect / @colby-swandale , ship whenever you think it's ready!

**Comment by indirect** [comment]
Some errands to run this morning, but I'll try to get this all shipped later this afternoon. 👍

**Comment by colby-swandale** [comment]
i unfortunately don't have access to staging/production to make a deploy.

**Comment by indirect** [comment]
Oh! This one's fully automatic... production is just the repo bundler/bundler.github.io, and "deploy" is just running `rake deploy` in this repo. But Travis automatically deploys the master branch to production now, I think. So here we go!

**Pull request merged**